quackosm/osm_extracts/_geojson_parser.py

```
@@ -0,0 +1,68 @@
+def parse_geojson_file(geojson_url: str) -> Optional["BaseGeometry"]:  # pragma: no cover
+    """
+    Parse a GeoJSON file from URL into a single geometry describing the region extent.
+
+    Args:
+        geojson_url (str): URL to load a GeoJSON file.
+
+    Returns:
+        Optional[BaseGeometry]: Parsed geometry, or `None` if the request returns 404 not found
+            or the file contains no geometry.
+    """
+    result = requests.get(geojson_url, headers={"User-Agent": _USER_AGENT})
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)